src/Engine/TranskribusEngine.php

```diff
@@ -0,0 +1,86 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Engine;
+use App\Exception\OcrException;
+use Krinkle\Intuition\Intuition;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class TranskribusEngine extends EngineBase
+{
+    /** @var string The API key. */
+    protected $key;
+
+    /** @var TranskribusClient */
+    protected $transkribusClient;
+
+    /**
+     * TranskribusEngine constructor.
+     * @param string $username of transkribus account.
+     * @param string $password of transkribus account.
+     * @param Intuition $intuition
+     * @param string $projectDir
+     * @param HttpClientInterface $httpClient
+     */
+    public function __construct(
+        string $username,
+        string $password,
+        Intuition $intuition,
+        string $projectDir,
+        HttpClientInterface $httpClient
+    ) {
+        parent::__construct($intuition, $projectDir, $httpClient);
+        $credentials = [
+            'username' => $username,
+            'password' => $password
+        ];
+        $this->transkribusClient = new TranskribusClient($httpClient, $credentials);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function getId(): string
+    {
+        return 'transkribus';
+    }
+
+    /**
+     * @inheritDoc
+     * @throws OcrException
+     */
+    public function getResult(
+        string $imageUrl,
+        string $invalidLangsMode,
+        array $crop,
+        ?array $langs = null
+    ): EngineResult {
+        $this->checkImageUrl($imageUrl);
+
+        [ $validLangs, $invalidLangs ] = $this->filterValidLangs($langs, $invalidLangsMode);
+
+        $image = $this->getImage($imageUrl, $crop);
+        $imageUrl = $image->getUrl();
+        $response = $this->transkribusClient->initProcess($imageUrl, []);
+
+        if ($response->getHasError()) {
+            file_put_contents('initp.json', "init process failed");
```

Comment:
What is this file for? It doesn't seem to be used anywhere.

Reply:
Will remove that, was trying to debug